input.ftd

```diff
@@ -43,4 +43,10 @@ Integer accepts the below attributes along with the
 - `id`
 - `placeholder`
 - `position`
+- `value`: sets the current value
+- `default-value`: sets the defaultValue Property with the specified value
+
```

Comment:
Refer other files eg https://ftd.dev/text/ to see how each argument is documented. Fix this entire file.